Applications/spire/include/spire/book_view/technicals_panel.hpp

```diff
@@ -14,6 +16,15 @@ namespace Spire {
       //! Constructs a TechnicalsPanel.
       TechnicalsPanel(QWidget* parent = nullptr);
 
+      //! Sets the model the panel gets its data from.
+      /*
+        \param model The associated model.
+      */
+      void set_model(std::shared_ptr<BookViewModel> model);
+
+      // Resets the text of all labels to their default values.
+      void reset_labels();
```

Comment:
Rename this to "reset_model" and implement it in such a way that it sets its own model to null and resets all labels.